### PR TITLE
Added proper inference & Fixed exploding gradients

### PR DESCRIPTION
**Pull request by Rick-McCoy**
Added inference for TTS

Fixed exploding gradients in attention function

Fixed major bug in dataloader for Tiers 2~: source and target were switched

Fixed miscellaneous bugs

### CODE REVIEW

**Comment by Rick-McCoy** [comment]
Now features ~2x more efficient training & inference time via PackedSequence.

Updated README.
